package.json

```diff
@@ -65,6 +65,9 @@
       "plugin:import/typescript"
     ],
     "rules": {
+      "@typescript-eslint/prefer-optional-chain": "off",
+      "@typescript-eslint/no-explicit-any": "off",
+      "@typescript-eslint/no-unsafe-member-access": "off",
```

Comment:
Once this rule can distiginguish between explicit `any` and implicit/external `any` we can disable. Until then its false-positive rate is too high.